docs/methodology/llm_inference.md

```diff
@@ -62,16 +62,22 @@ We fit a linear regression model to the dataset, which models the energy consump
   <figcaption>Figure: Energy consumption (in Wh) per output token vs. number of active parameters (in billions)</figcaption>
 </figure>
 
-The linear regression gives
+??? info "What is a 95% confidence interval?"
+
+    The standard deviation $\delta$ of a linear regression measures "how close are the datapoints to the fitted line". A priori, the larger it is, the worse is the approximation. Consider a linear regression $Y(x) \approx \alpha x + \beta$. Given some assumptions (see [here](https://en.wikipedia.org/wiki/Linear_regression)), we can say that $Y(x) \in [\alpha x + \beta - 1.96\delta, \alpha x + \beta + 1.96\delta]$ with probability 95%.
```

Comment:
Maybe we should cite this as well for the 1.96 approximation (https://en.wikipedia.org/wiki/97.5th_percentile_point). Or any other better explanation / course.